actions/actionsRouter.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const db = require('../data/helpers/actionModel');
+const middleware = require('../common/middleware');
+
+const router = express.Router();
+const {ensureValidAction, ensureValidActionId} = middleware;
+
+router.get('/', (req, res) => {
+  db.get()
+    .then(actions => {
+      res.status(200).json(actions);
+    })
+    .catch(err => {
+      res.status(500).json({
+        errorMessage: `there was an error retrieving the actions: ${err}`,
+      });
+    });
+});
+
+router.get('/:id', ensureValidActionId, (req, res) => {
+  // id will be checked in middleware. if we get here, send the project
+  db.get(req.params.id)
```

Comment:
Could save the `req.params.id` into a variable if you wanted to, after hearing Luis talk about magic numbers I'm pretty sure you'd want to do that here too. Something like `const { id } = req.params`. (although the req.params.id is obviously an id so it does seem a little redundant)